bucket/paperwork.json

```diff
@@ -3,8 +3,12 @@
     "description": "Personal document manager",
     "homepage": "https://openpaper.work",
     "license": "GPL-3.0-or-later",
-    "url": "https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip",
-    "hash": "559815cdfeb5e62c4ffcbfe41ab3986c3e30d1bc1c63179da9ef80886d0872bd",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.openpaper.work/windows/amd64/paperwork-master-latest.zip",
```

Comment:
I don't know if there is one, but it would be nice to have a download address with a version number, as this would prevent the manifest file from constantly reporting errors due to hash changes.